Cargo.toml

```diff
@@ -39,12 +39,12 @@ tokio = { version = "~1.17", features = ["full"] }
 actix-web = { version = "4.0.1", optional = true }
 tonic = "0.7.1"
 num-traits = "0.2.14"
-prost = "0.10.1"
-prost-types = "0.10.1"
 
-raft = { version = "0.6.0", features = ["prost-codec"], default-features = false, optional = true }
+# Consensus related crates
+raft = { version = "=0.6.0", features = ["prost-codec"], default-features = false, optional = true }
 slog =  { version = "2.7.0", optional = true }
 slog-stdlog = { version = "4.1.1", optional = true }
+prost = { version = "=0.7.0", optional = true }
```

Comment:
did it fail with `prost` 0.10.1?

Reply:
Yes, `raft-rs` uses prost `0.7.0` so basically there is no other way to serialize raft messages